src/state/connection-mixin.ts

```diff
@@ -275,6 +276,23 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
       subscribeFrontendUserData(conn, "core", (userData) =>
         this._updateHass({ userData })
       );
+
+      const pingBackend = async () => {
+        if (this.hass?.connected) {
+          promiseTimeout(5000, this.hass?.connection.ping()).catch(() => {
+            if (!this.hass?.connected) {
+              return;
+            }
+
+            // eslint-disable-next-line no-console
+            console.log("Websocket died, forcing reconnect...");
+            this.hass?.connection.reconnect(true);
+          });
+        }
+        setTimeout(pingBackend, 5000);
```

Comment:
Should the interval be higher? In this case the function can probably just be scheduled with setInterval

Reply:
To know if the interval should be higher we really need to test or know when the issue occurs...

Reply:
My own experience with the bug is that a few minutes are enough to trigger